exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/ApiControllerTest.java

```diff
@@ -164,56 +159,62 @@ public void submitTransactionWhenInternalServerErrorIsThrown(TestContext context
     post(ApiController.SUBMIT_TRANSACTION_PATH)
         .sendJsonObject(
             new JsonObject(messageJson),
-            context.asyncAssertSuccess(ar -> {
-              context.verify(v -> {
-                assertThat(ar.statusCode()).isEqualTo(HTTP_INTERNAL_ERROR);
-                verify(service).submitTransaction(transaction);
-              });
-            }));
+            context.succeeding(ar -> context.verify(() -> {
```

Comment:
`ar` is confusing

Reply:
[async](Result|Response) — migrated to result/response where they are actually used.